app/lib/personal_for_you.rb

```diff
@@ -3,30 +3,21 @@
 class PersonalForYou
   include Redisable
 
-  FOR_YOU_OWNER_ACCOUNT = ENV['FOR_YOU_OWNER_ACCOUNT'] || 'admin'
-  BETA_FOR_YOU_LIST = 'Beta ForYou Personalized'
   ACCOUNT_RELAY_AUTH = "Bearer #{ENV.fetch('ACCOUNT_RELAY_KEY')}"
   ACCOUNT_RELAY_HOST = 'acctrelay.moth.social'
 
-  def beta_list_accounts
-    default_owner_account = Account.local.where(username: FOR_YOU_OWNER_ACCOUNT).first!
-    beta_list = List.where(account: default_owner_account, title: BETA_FOR_YOU_LIST).first!
-    beta_list.accounts.without_suspended.includes(:account_stat)
-  end
-
   # Indirect Follows are the following of your followers
+  # Get full account handle <example@moth.social>
   # IE Friends of Friends
-  def statuses_for_indirect_follows(account)
-    # Get full account handle <example@moth.social>
-    account_handle = account.local? ? account.local_username_and_domain : account.acct
```

Comment:
Is this no longer needed (so we always are getting the full account handle here, right?)

Reply:
yeah since we're pulling from account relay the user's handle, rather than their local account model from the Moth.Social db, we can skip the check and use it directly yeppers.